libs/commands/init/src/command.ts

```diff
@@ -27,6 +27,10 @@ const command: CommandModule = {
       type: "boolean",
       default: false,
     },
+    skipInstall: {
+      describe: "Skip installation of npm dependencies after initialization",
```

Comment:
```suggestion
      describe: "Skip installation of dependencies after initialization",
```

Seemed too specific in light of the features this PR itself is adding!